README.md

```diff
@@ -270,11 +270,35 @@ and nested attributes, you may wish to write your own changes method that includ
 
 Mongoid::History provides an option named `:changes_method` which allows you to do this.  It defaults to `:changes`, which is the standard changes method.
 
+Note: When your changes method will provide a field key that is not actually a field on the model, that will need to be added to the list of fields in the
```

Comment:
How about this:

"Specify additional fields that are provided with a custom `changes_method` with the `:on` option. "

Lets not shout :) 

"To specify current fields and additional fields use `fields.keys + [:custom]`."